src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -931,5 +932,107 @@ public void WorkingDirectory_Set_GetReturnsExpected(string workingDirectory)
             var info = new ProcessStartInfo { WorkingDirectory = workingDirectory };
             Assert.Equal(workingDirectory ?? string.Empty, info.WorkingDirectory);
         }
+
+        [Fact(Skip = "Manual test")]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]
+        public void StartInfo_WebPage()
+        {
+            ProcessStartInfo info = new ProcessStartInfo
```

Comment:
I wonder if we should a property to FileCleanupTestBase called InteractiveTests for this. It would be nice to have an established pattern.  I'll follow up in a separate PR.